__tests__/cypress.config.js

```diff
@@ -0,0 +1,3 @@
+import configure from '@heartexlabs/ls-test';
```

Comment:
Right now there is a problem to run it on Window 10. 
https://github.com/heartexlabs/ls-frontend-test/blob/main/configure.js#L11
`process.env.PWD` is `undefined`

Reply:
thx, will take a look into it. as a workaround for now you can try running it under WSL. unfortunately, I don't have windows right now to check it

Reply:
oh, it looks like we can replace it with `process.cwd()` instead

Reply:
In my case works
```JavaScript
const localPath = p => path.resolve(process.cwd(), p);
```
but I don't know if it is cross platform.